docs/operations/breaking-changes.md

```diff
@@ -2,43 +2,43 @@
 
 Evolving the Lockbook code base requires some special considerations. 
 
-## Breaking Changes Between `core` Versions
+## Breaking Changes Between Core Versions
 
-Our clients store and maintain local state due to the fact that we support offline use. If we update our app, we need to make sure the new version of the app can gracefully interact with the current data on the device. Ultimately, we can always nuke the local state and re-sync with the server. However, we want to avoid doing this where possible for the following reasons:
+Our clients store and maintain local state because we support offline use. If we update our app, we need to make sure the new version of the app can gracefully interact with the current data on the device. Ultimately, we can always nuke the local state and re-sync with the server. However, we want to avoid doing this where possible for the following reasons:
 
 + We would need to explicitly keep track of offline changes that haven't yet been synced
 + Depending on the user this could take a long time
-+ Needlessly places a load our infrastructure
++ Places needless load on our infrastructure
 
-Therefore, where possible, we should support old functionality. When this is not possible (a data format changed), a "migration" must be performed. Encoded in rust forever will be instructions on how to step through each version of state. These migrations should not require an internet connection. All core endpoints that interact with state may return `MigrationRequired`. You can also explicitly check the status by calling `get_db_state`.
+Therefore, where possible, we should support old functionality. When this is not possible (data format changed), a "migration" must be performed. Encoded in Rust forever will be instructions on how to step through each version of the local state. These migrations should not require an internet connection. All core endpoints that interact with the local state may return `MigrationRequired`. You can also explicitly check the status by calling `get_db_state`.
```

Comment:
We don't return `MigrationRequired` anymore, we just do it in `init`. Possibly still doc worthy.